File: 1598-crawler-log-folder/1598-crawler-log-folder.cs
public class Solution {
    public int MinOperations(string[] logs) {
        var depth = 0;
        foreach(var cmd in logs){
            if(cmd.Equals("../") && depth > 0)
                depth--;
            else if(cmd[0] != '.')
                depth++;
        }
        return depth;
    }
}

File: 1598-crawler-log-folder/1598-crawler-log-folder.cpp
class Solution {
public:
    int minOperations(vector<string>& logs) {
        int depth = 0;
        for(string cmd : logs){
            if(cmd == "../" && depth > 0)
                depth--;
            else if(cmd[0] != '.')
                depth++;
        }
        return depth;
    }
};